.github/workflows/gradle-build.yml

```diff
@@ -0,0 +1,34 @@
+name: Gradle Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+          server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+          settings-path: ${{ github.workspace }} # location for the settings.xml file
+
+      - name: Publish
+        run: ./gradlew publishToMavenLocal --console=plain
+        
+      - name: Run tests
+        run: |
+          cd tests
+          ./gradlew test --console=plain
+          cd ..
```

Comment:
Maybe` ./tests /gradlew test --console=plain` is better?

Reply:
This did not work for me, but I think adding working-directory is also fine.